README.md

```diff
@@ -4,7 +4,7 @@
 
 Follow the instructions below to set up a Quart application in your environment:
```

Comment:
@k-allagbe I find the README.md extremely confusing. I suggest we break it down into separate docs to have a short and clean README.md. Should be done in a separate issue/PR

Reply:
Agreed. https://github.com/ai-cfia/membrane-backend/issues/83